src/components/AuthBar.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+
+import { AuthService  } from "../machines/authMachine";
+
+export const AuthBar = () => {
+    const { ready, dispose, signOut } = AuthService();
+
+    useEffect(() => {
+        ready(); // lift this up - you'd want one auth service shared across a platform
```

Comment:
`<AuthenticatedContainer />` for the use effect, maybe even to create a context or is that just extra bloat?

```
authenticated: boolean;
signInUser: (userData) => void;
signOutUser: () => void;
```